### PR TITLE
Mobile empty states

### PR DESCRIPTION
**Pull request by axel-denis**
Fixes ARE-397

### CODE REVIEW

**Review by Tomi-Tom — COMMENTED** [review]
LGTM

File: frontend/mobile/lib/pages/library.dart [hunk 1]
```diff
@@ -31,34 +36,81 @@ class _LibraryState extends State<Library> {
...
-            },
+                  } else {
+                    if (needRefresh) {
+                      needRefresh = false;
+                      if (snapshot.data!.data != null) {
+                        initialWorkflows = snapshot.data!.data!;
+                        workflows = snapshot.data!.data!;
+                      }
+                    }
+                    if (workflows.isEmpty) {
```
Comment (RezaRahemtola) [inline frontend/mobile/lib/pages/library.dart:89]: add a more specialized message here ("no workflows found" for example)

File: frontend/mobile/lib/pages/library.dart
Comment (RezaRahemtola) [inline frontend/mobile/lib/pages/library.dart:1]: remove the search bar changes from this PR

Comment (axel-denis) [reply frontend/mobile/lib/pages/library.dart:1]: will be very complicated to remove things like this

Comment (RezaRahemtola) [reply frontend/mobile/lib/pages/library.dart:1]: just start your branch from main and add your empty states changes to it (cherry pick)

**Pull request merged**